specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/examples/PlaybookConfigurations/GetPlaybookConfigurationResourceGroup_example.json

```diff
@@ -0,0 +1,52 @@
+{
+  "parameters": {
+    "api-version": "2019-01-01-preview",
+    "subscriptionId": "20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+    "resourceGroupName": "myRg",
+    "playbookConfigurationName": "myPlaybookConfigName"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/myRg/providers/Microsoft.Security/playbookConfigurations/myPlaybookConfigName",
+        "name": "myPlaybookConfigName",
+        "type": "Microsoft.Security/playbookConfigurations",
+        "properties": {
+          "description": "Alerts auto dismiss",
+          "isEnabled": true,
+          "metadata": {
+            "createdDateTimeUtc": "2019-03-27T08:43:54.5755094Z",
+            "createdBy": "2c429ed8-089f-42aa-9bb9-92312b8c9f72",
+            "lastUpdatedDateTimeUtc": "2019-03-28T08:43:54.5755094Z",
+            "lastUpdatedBy": "2c429ed8-089f-42aa-9bb9-92312b8c9f72"
+          },
+          "scopes": [
+            "/subscriptions/6618f01b-9b61-4cfc-9010-836b34b1bb2a",
+            "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23"
+          ],
+          "sources": [
+            {
+              "eventSource": "AscAlerts",
+              "ruleSets": [
+                {
+                  "rules": [
+                    {
+                      "propertyJPath": "AlertType",
+                      "propertyType": "System.String",
```

Comment:
please use a standard or your own enum
not all of our customers are .Net developers 😄

Reply:
:) I was thinking of exposing a specific list via documentation, rather than an enum here to avoid continuous modifications (the same as in Compute [e.g.](https://github.com/Azure/azure-rest-api-specs/blob/6081866424e897b5ba7890bdb7279f4a02e481e6/specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json#L854) ). What's better?